spec/controllers/users_controller_spec.rb

```diff
@@ -112,7 +112,20 @@
         end
 
         it 'should save the user' do
-          expect(User.find(assigns(:user))).not_to be_nil
+          expect(User.find(assigns(:user).id)).not_to be_nil
```

Comment:
@orenyk - This reoccurs a lot, wouldn't it be better practice to create a "it behaves like"?